training/drive_to_ball_grader.py

```diff
@@ -0,0 +1,32 @@
+from dataclasses import dataclass, field
+from math import sqrt
+from typing import Optional
+
+from rlbot.training.training import Grade, Pass, Fail
```

Comment:
This is intended; however I'm open to have my mind changed.
The training support code in rlbot is prone changes in the internals of rlbot (SetupManager/GameInterface). We have unit tests in rlbot which would fail if someone were to break the required stuff for rlbottraining, which would be tricky to discover (similar problems to when I broke the new GUI with the `num_players` refactor)

These sorts of thing seems to be a rlbot responsibility rather than a rlbottraining responsibility:
https://github.com/RLBot/RLBot/blob/master/src/main/python/rlbot/training/training.py#L196
https://github.com/RLBot/RLBot/blob/master/src/main/python/rlbot/training/training.py#L226

We probably wouldn't need this adapter class if we put everything into rlbottraining.
https://github.com/RLBot/RLBotTraining/blob/master/rlbottraining/training_exercise_adapter.py

Maybe I should re-export the Grade/Pass/Fail within rlbottraining since it would seem odd to a user to import from both packages about training?

Reply:
That rationale for keeping it in RLBot sounds good to me! Regarding the re-export of Grade/Pass/Fail, I don't have a strong opinion about that.